mix.exs

```diff
@@ -4,8 +4,8 @@ defmodule AwsSsmProvider.MixProject do
   def project do
     [
       app: :aws_ssm_provider,
-      version: "0.2.1",
-      elixir: "~> 1.7",
+      version: "0.3.0",
+      elixir: "~> 1.8",
```

Comment:
any reason not to go all the way to 1.9?

Reply:
I'd recommend against it because it would be incompatible with many who are probably still running on 1.8 (like CareDox!)